app/views/exports/pbmc_bank_tubes_content_report.csv.erb

```diff
@@ -0,0 +1,83 @@
+<%# This is a report of the contents of the tubes in the PBMC Bank plate %>
+<%# Based on app/views/exports/hamilton_lrc_pbmc_bank_to_lrc_bank_seq_and_spare.csv.erb %>
+<%= CSV.generate_line ['Workflow', @workflow], row_sep: '' %>
+
+<%=
+  CSV.generate_line [
+                      'Well name',
+                      'Donor ID',
+                      'Parent barcode',
+                      'FluidX barcode',
+                      'Extraction and freeze date',
+                      'Sequencing or contingency',
+                      'Cell count (cells/ml)',
+                      'Viability (%)',
+                      'Volume (µl)',
+                      'Study name',
+                      'Collection site'
+                    ],
+                    row_sep: ''
+%>
+<%
+  rows_array = []
+  @plate.wells_in_columns.each do |source_well|
+    # skip if well empty (not all wells have samples in them)
+    next if source_well.empty?
+
+    # skip if well not in passed state
+    next unless source_well.passed?
+
+    # skip if no downstream tubes for this well (not set up children in Limber yet)
+    next if source_well.downstream_tubes.empty?
+
+    child_tube_v1 = source_well.downstream_tubes.last
+    child_tube_v2 =
+      Sequencescape::Api::V2.tube_with_custom_includes(
+        'custom_metadatum_collection',
+        nil,
+        barcode: child_tube_v1.labware_barcode.machine
+      )
+
+    next unless child_tube_v2 && child_tube_v2.custom_metadatum_collection
+
+    # get tube metadata from child_tube
+    metadata = child_tube_v2.custom_metadatum_collection.metadata
+
+    next unless metadata
+
+    tube_rack_barcode = metadata['tube_rack_barcode']
+    tube_rack_position = metadata['tube_rack_position']
+
+    next unless tube_rack_barcode && tube_rack_position
+
+    sample_uuid = source_well.aliquots.first.sample.uuid
+    source_well_posn = source_well.position['name']
+
+    # e.g. SEQ:CHILD_TUBE:A1
+    child_tube_name_array = child_tube_v2.name.split(':')
+
+    sample_metadata = source_well.aliquots.first.sample.sample_metadata
+
+    # sourced from deep behind sample manifest UAT action on the SS side
```

Comment:
This comment seems like it's only relevant to a test environment, as it references UAT actions (which are not used in prod)?